### DISCUSSION TITLE
712

### DISCUSSION THREAD

**Discussion by giscus** [jojoldu/blog-comments#902] [Comments]
# 712

365/24 로 관리하는 시스템에서 로그는 굉장히 중요하다. 하지만 로그가 중요하다는 생각에 무분별하게 남기는 것은 좋지 않다. 대표적인 예로 습관적으로 예외 상황이 발생하면 ERROR 레벨로 로그를 남기는 경우이다. 보통의 서비스에서는 시간 내 에러 로그가 일정 수치 이상 쌓이면 알람을 발생시키도록 구성한다. 정상적이지 않은 모든 상황에서 전부 ERROR 레벨로 처리하게 되면 불필요하게 많은 알람들로 인해 정작 봐야할 심각한 에러 로그들도 놓칠 수 있다. 그래서 적정 수준에서 로그 레벨을 구분하여 알람 경보 수준도 구분하는 것이 필요하다. 1. 로그 레벨 로그 레벨은 해당 로그 메세지가 얼마나 중요한지를 알려주는 정보이다. 로그 레벨의 중요도는 담당 개발자가 밤에 계속 잠을 잘 수 있는지, 즉시 침대에..

https://jojoldu.tistory.com/712

**Comment by hongji3354** [comment]
마지막 내용중 특정 개수일 때 알람오는 거는 어떤 서비스를 사용하고 계신지 알 수 있을까요?

```
INFO: 기존대비 +-50%이상 차이날 경우 알람
WARN: 분당 20개이상일 경우 알람
ERROR: 분당 5개이상일 경우 알람
```

**Comment by khyunji99** [comment]
좋은 글 감사합니다!
